src/traffic_cache_tool/CacheTool.cc

```diff
@@ -457,7 +459,7 @@ Cache::loadSpan(FilePath const &path)
 {
   Errata zret;
```

Comment:
Why not this `zret` too?

Reply:
I only assign the global err in the handlers of each command. I left these helper methods the way they are.